e2e-tests/capabilities-add-drop-build.yaml

```diff
@@ -21,17 +21,17 @@ pipeline:
   - name: Test default effective capability
     runs: |
       # Skip test if on QEMU runner, since the runner does not support process capabilities add/drop Melange feature.
-      { test -f /sys/class/dmi/id/sys_vendor && grep -E "^QEMU$" /sys/class/dmi/id/sys_vendor && exit 0; } || true
+      {  grep -E "melange_qemu_runner=1" /proc/cmdline && exit 0; } || true
```

Comment:
this would be much more clearly written as:

```
grep "melange_qemu_runner=1" /proc/cmdline && { echo "SKIP - running inside melange runner"; exit 0; }
```